### ISSUE TITLE
Clear the counts in the history when data changes

### ISSUE THREAD

**Issue by kimrutherford** [pombase/website#499]
The counts are stored in the history.  We should clear them when there is a data update.  That means the browser code needs to query the server occasionally to check when a new version is available.

**Comment by kimrutherford** [comment]
As a temporary measure the counts are now reset if you reload.  So can we delay checking for data changes until after the release?  I'm not sure of the best way to do it yet.

**Comment by ValWood** [comment]
Yes, this should be high priority after live though. It's pretty annoying, but probably only for me ;)
I "kind of  recognise" which query is which based on the count rather than the query string (which is harder to process), so i look down the column to  see which on I think it is, and then check the query string.


![queries](https://user-images.githubusercontent.com/7359272/30314903-18f94aa6-979a-11e7-83c7-0196762d8b28.jpg)

**Comment by ValWood** [comment]
so in the above, most will not change very often. In fact most totals don't change very often. Even "slim terms" are relatively stable now, but features like  protein length and tmm number only change in a blue moon.

**Comment by kimrutherford** [comment]
I've made a quick hack that might help.  If you reload the preview you'll be able to double-click on the "Count" header to recalculate all the counts.  We could make that a button if you think it would be useful to others.

**Comment by ValWood** [comment]
yes, lovely

**Comment by kimrutherford** [comment]
I realised that we could do the same recalculation on every reload.  So now I've changed it so you don't need to double click on the header any more.  It will happen automatically when you start the browser or reload the page.

**Comment by ValWood** [comment]
This is OK for release.

LAter It would be better if the counts satyed, *except* when the totla change (which makes it clear which things have updated_

**Comment by kimrutherford** [comment]
I think we can close this.  I've changed the code to restore the result counts from the previous session (rather than showing "Results...").  There was a fix few weeks ago that reruns all the queries when you reload.  So now the counts quietly fix themselves.

**Comment by ValWood** [comment]
because lots of counts are fairly stable, and will probably *never* change, but some will change, I though it would be quite nice in the longer term to see the counts which did not change, but if the number had changed the column would say "rerun query"

This would enable users to see at a glance which query totals had changed since they last looked.

Is this possible longer term?

**Comment by kimrutherford** [comment]
> Is this possible longer term?

Yep, I think so.

**Comment by kimrutherford** [comment]
> I though it would be quite nice in the longer term to see the counts which did not change, but if the number
> had changed the column would say "rerun query"

I've had a go at implementing this.  I'm about to update pombase-dev.  It currently looks like this.  How would you like it to look?  If you hover over the red text it says "Click to re-run this query".

![out_of_date_results-2018-01-18](https://user-images.githubusercontent.com/90474/35082230-823d19a0-fc7d-11e7-8015-cc2fe078b38f.png)

**Comment by ValWood** [comment]
I think that is great...It will really indicate what has changed!

**Comment by kimrutherford** [comment]
This should be live on the main site now.

**Comment by kimrutherford** [comment]
If that's working should we close this issue?
